nexusadspy/segment.py

```diff
@@ -0,0 +1,100 @@
+
+from __future__ import (
+    print_function, division, generators,
+    absolute_import, unicode_literals
+)
+
+from nexusadspy.client import AppnexusClient
+from io import BytesIO
+from gzip import GzipFile
+import time
+
+
+class SegmentsUploader:
+    """
+    batch-upload API wrapper for appnexus
```

Comment:
move this docstring to the docstring in __init__. avoid mixing class-level and constructor-level docstrings